app/Components/Typhography/Typhography.tsx

```diff
@@ -0,0 +1,22 @@
+import { ColorsEnum } from "@/app/Enums/colors.enum"
+import { ReactNode } from "react"
+import styles from './Typhography.module.scss'
+import { TyphographyEnum } from "@/app/Enums/Typhography.enum"
+
+type Props = {
```

Comment:
make this interface and take it to its owny file Components/Typography/interfaces/typography-props.interface.ts